src/sentry/static/sentry/app/components/group/inboxBadges/inboxReason.tsx

```diff
@@ -30,6 +30,42 @@ function InboxReason({inbox, fontSize = 'sm'}: Props) {
       ? `More than ${Math.round(count / EVENT_ROUND_LIMIT)}k`
       : `${count}`;
 
+  const getToolTipDescription = ({
+    until,
+    count,
+    window,
+    user_count,
+    user_window,
+  }: InboxReasonDetails) => {
+    if (until) {
+      // Was ignored until `until` has passed.
+      //`until` format "2021-01-20T03:59:03+00:00"
+      return t('Was ignored until %(window)s', {
+        until,
+      });
+    } else if (count) {
+      // Was ignored until `count` events occured
+      // If `window` is defined, than `count` events occurred in `window` minutes.
+      // else `count` events occured since it was ignored.
+      return t('Was ignored until it occured %(count)s time(s)%(window)s', {
+        count: getCountText(count || 0),
+        window: window ? ' in ' + getDuration((window || 0) * 60, 0, true) : '',
+      });
+    } else if (user_count) {
+      // Was ignored until `user_count` users were affected
+      // If `user_window` is defined, than `user_count` users affected in `user_window` minutes.
+      // else `user_count` events occured since it was ignored.
+      return t('Was ignored until it affected %(user_count)s user(s)%(user_window)s', {
+        user_count: getCountText(user_count || 0),
+        user_window: user_window
+          ? ' in ' + getDuration((user_window || 0) * 60, 0, true)
+          : '',
+      });
+    } else {
+      return ''; // Shouldn't happen, but wasn't sure of a safe fallback.
```

Comment:
```suggestion
    if (until) {
      // Was ignored until `until` has passed.
      //`until` format "2021-01-20T03:59:03+00:00"
      return t('Was ignored until %(window)s', {
        until,
      });
    } 
    
    if (count) {
      // Was ignored until `count` events occured
      // If `window` is defined, than `count` events occurred in `window` minutes.
      // else `count` events occured since it was ignored.
      return t('Was ignored until it occured %(count)s time(s)%(window)s', {
        count: getCountText(count || 0),
        window: window ? ' in ' + getDuration((window || 0) * 60, 0, true) : '',
      });
    }
    
      if (user_count) {
      // Was ignored until `user_count` users were affected
      // If `user_window` is defined, than `user_count` users affected in `user_window` minutes.
      // else `user_count` events occured since it was ignored.
      return t('Was ignored until it affected %(user_count)s user(s)%(user_window)s', {
        user_count: getCountText(user_count || 0),
        user_window: user_window
          ? ' in ' + getDuration((user_window || 0) * 60, 0, true)
          : '',
      });
    } 
    
   return ''; // Shouldn't happen, but wasn't sure of a safe fallback.
```

Reply:
Thanks! Committed this suggestion and I'll change the final return to account for Taylan's suggestion.